packages/sanes-chrome-extension/src/extension/backgroundscript.ts

```diff
@@ -22,30 +26,53 @@ chrome.runtime.onMessage.addListener(
         // do an operation in the extension
         // replying back to content script
         break;
+      case 'create_persona':
```

Comment:
can you create and use a constant for this 'create_persona'?

Reply:
The unit type `readonly action: 'ctb_HELLO' | 'create_persona';` in `MessageToBackground` serves as a constant. Any other string value than those are rejected by the type system.

Reply:
I know that but... I am using to use the finder to find by references in order to allocate things faster. I think it is good for doing a brain connection of parts involved in the app.

I can do that by searching by string, of course, but is less... elegant, isn't it?